### PR TITLE
Categorize Countries

### PR DESCRIPTION
**Pull request by MAChitgarha**
See this: #575

### CODE REVIEW

**Comment by MAChitgarha** [comment]
@pesehr Would it be merged?

**Comment by ruchern** [comment]
There needs to be a way to dynamically sort this.

**Comment by MAChitgarha** [comment]
@ruchern Yes. Maybe using Travis CI to run a script every time and pushing back here? If yes, I would write the script to automate the process.

**Comment by ruchern** [comment]
I see more of making use of Github pages, and create a page to recognise all contributors. Using Javascript, you should be able to sort them accordingly.

**Comment by pi0** [comment]
I don't like the idea of categorizing. The idea is GitHub is for everyone. Regardless of country.

**Comment by ruchern** [comment]
Author probably just want to make the page looks cleaner and easier to read.

**Comment by MAChitgarha** [comment]
@pi0 We don't want to show that GitHub is not for everyone. We only want to categorize it to look and feel better, and everyone can see which countries has supported us.

**Comment by MAChitgarha** [comment]
@pi0 And adding supporters and maintaining them will be easier.

**Comment by MAChitgarha** [comment]
@ruchern The language of the script does not matter, however. The acceptance of this pull request is more important. I (or others) may add an automation script later.

**Comment by MAChitgarha** [comment]
@pi0 And we can mention in the supporters section that we categorized by country and ordered by supporters count of each one to prevent misunderstandings.

**Comment by chadidi** [comment]
@pi0 @MAChitgarha Can you guys merge this so we can add our support 🙌.

**Comment by ruchern** [comment]
I am not referring to your Travis CI script. More of using Github Pages to do up a support list.

**Comment by MAChitgarha** [comment]
@chadidi We need a collaborator to do that. I'm not a collaborator, sorry! However, add your name to the list of supporters, and I'll fix the conflicts.

**Comment by MAChitgarha** [comment]
@ruchern GitHub pages is a good idea, but then how to accept new supporters? Via GitHub pages or something else?

**Comment by ruchern** [comment]
Yeah. Same using pull requests.

**Comment by MAChitgarha** [comment]
@ruchern I think the creator and collaborators can only create GitHub pages. So, we should wait for them.

**Comment by 1995parham** [comment]
@MAChitgarha I think we can wait until tonight then I think we can sort all supporters and create a timeline for this event if all contributors agree.

**Comment by MAChitgarha** [comment]
@1995parham All contributors or most of them? By the way, if you know about PHP life-cycle, [RFC](https://wiki.php.net/rfc)s will be agreed if more than 50% of contributors agree an RFC. However, it's all up to you.

**Comment by 1995parham** [comment]
@MAChitgarha I think most of them option is going to work for us.

File: README.md [hunk 1]
```diff
@@ -20,9 +21,9 @@ GitHub used to be an open and free platform for everyone, but it has decided to
 Eventually, we are hoping that GitHub's team:
 
 - Let us download our private repositories and also make them public (setting control).
-  - **UPDATE:** It is possible now to make private repositories public.
+   - **UPDATE:** It is possible now to make private repositories public.
 - Change their restriction policies to not limiting users based on their Nationality.
-- Announce their decision publicly and apologize for making this situation (like what slack did before in the same situation: https://slackhq.com/an-apology-and-an-update).
+- Cnnounce their decision publicly and apologize for making this situation (like what slack did before in the same situation: https://slackhq.com/an-apology-and-an-update).
```
Comment (akhilome) [inline README.md]: Slight Typo here &rarr; `Cnnounce`

**Comment by MAChitgarha** [comment]
@1995parham It's is able to be merged. Agreeing by contributors?

**Comment by chadidi** [comment]
@MAChitgarha I took a look, some names are missing including my own 😅.

**Comment by MAChitgarha** [comment]
@chadidi I would merge it with master again. Sorry for that. And if contributors won't pay attention to this pull request, I'll not continue it, because it's a bit hard task to merge names.

**Comment by chadidi** [comment]
I created this pull request https://github.com/1995parham/github-do-not-ban-us/pull/936 from the current master do you think it's better this way.

**Comment by ruchern** [comment]
Merge this and get the initial sorting. People might realise there is a sorting or categorisation and adhere to it.

**Comment by MAChitgarha** [comment]
@ruchern Reading from markdown is not hard, neither easy. And sorting by people name is not a good idea at all. It makes contributions a bit harder.

**Comment by MAChitgarha** [comment]
But I think the idea of categorizing names by countries and pushing poeple to the end of each country list would make things easier.

To prevent the misconception of not being equals, instead of ordering countries by number of contributors, we can sort them alphabetically.

**Comment by MAChitgarha** [comment]
Sorry to say, but this pull request would get closed in hours, without any attention from collaborators/contributors.

**Pull request closed without merging**